test/elixir/test/search_test.exs

```diff
@@ -71,7 +71,7 @@ defmodule SearchTest do
 
     url = "/#{db_name}/_design/inventory/_search/fruits"
     resp = Couch.get(url, query: %{q: "*:*", include_docs: true})
-    assert resp.status_code == 200
+    retry_until(fn -> resp.status_code == 200 end)
```

Comment:
Maybe we can do on extra single assert below, just to ensure it's not flapping and once it's hits 200 it stays 200

Reply:
How does retry_until work? Do it have any magic to throw an exception and the whole test body up to that point is retried. Otherwise, if it's as simple as it looks in `retry_until(fn -> resp.status_code == 200 end)` we'd just be checking the same response and always fail, then we'd have to retry the whole `Couch.get(...)`?